content/backstage/plugins/sonarqube.md

```diff
@@ -46,14 +46,15 @@ gettingStarted:
   - intro: Add the proxy config for SonarCloud
     language: YAML
     code: |
-      // app-config.yaml
+      # app-config.yaml
       proxy:
         '/sonarqube':
           target: https://sonarcloud.io/api
           allowedMethods: ['GET']
           headers:
-            Authorization: # Content: 'Basic base64("<api-key>:")' <-- note the trailing ':' # Example: Basic bXktYXBpLWtleTo=
-              $env: SONARQUBE_AUTH_HEADER
+            Authorization: Basic ${SONARQUBE_AUTH}
+            # Content: 'base64("<api-key>:")' <-- note the trailing ':'
+            # Example: bXktYXBpLWtleTo=
```

Comment:
Could you take these steps and explain to the user how to execute them in the notes section below?

I would like to add some value for the reader, over and above what is already present in the NPM page & README.

Personally I don't understand what the comments above are trying to tell me to do, so this might be a good chance to simplify for the reader and add some value.

Reply:
Ok cool I'll try come up with something